notifications_utils/field.py

```diff
@@ -68,7 +68,7 @@ class Field:
     def __init__(
         self,
         content: str,
-        values: Dict[str, Any] = None,
```

Comment:
ah, what a joke Python typing is. Declare something a dict, then immediately set it to `None` and there's no problem. 🤦